dm/ctl/master/query_status.go

```diff
@@ -60,5 +74,49 @@ func queryStatusFunc(cmd *cobra.Command, _ []string) {
 		return
 	}
 
-	common.PrettyPrintResponse(resp)
+	if resp.Result && taskName == "" && len(workers) == 0 {
+		result := wrapTaskResult(resp)
+		common.PrettyPrintInterface(result)
+	} else {
+		common.PrettyPrintResponse(resp)
+	}
+}
+
+// wrapTaskResult pick task info and generate tasks' status and relative workers
+func wrapTaskResult(resp *pb.QueryStatusListResponse) *taskResult {
+	taskStatusMap := make(map[string]string, len(resp.Workers))
+	taskCorrespondingWorkers := make(map[string][]string, len(resp.Workers))
+	for _, worker := range resp.Workers {
+		for _, subTask := range worker.SubTaskStatus {
```

Comment:
do we need to consider the status of _relay_ unit when _sync_ is the current process unit?

Reply:
The definition of task's status has been updated in PR description. The code has been revised correspondingly.